Peer.Domain/Models/PeerPullRequest.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace Peer.Domain.Models
+{
+    public class PeerPullRequest
+    {
+        private string _title;
+        private object _assignee;
+
+        public PeerPullRequest(string Title, object Assignee)
+        {
+            _title = Title;
+            _assignee = Assignee;
+        }
+
+        public string Title { get; set; }
```

Comment:
It looks like you're doing an auto implemented property here but you assign the values to private props. This doesn't work 😟 you can get rid of lines 5/6 and just assign to Title in the ctor.

Reply:
This was a typo. Forgot to clean up. Thanks for pointing it out.